drain3/drain.py

```diff
@@ -419,3 +419,231 @@ def get_total_cluster_size(self):
         for c in self.id_to_cluster.values():
             size += c.size
         return size
+
+
+class JaccardDrain(Drain):
```

Comment:
Please split this class into another file.

Reply:
Okay, that's done